.coderabbit.yml

```
@@ -0,0 +1,60 @@
+  collapse_ellipsis: false
+  auto_reply: true
+  
+chat:
+  auto_reply: true
+
+# Python-specific configuration
+python:
+  # Set the Python path to include backend/src
+  pythonpath:
+    - "backend/src"
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)